WordPress/src/main/java/org/wordpress/android/ui/main/SitePickerActivity.java

```diff
@@ -205,6 +204,17 @@ protected void onStart() {
 
     @SuppressWarnings("unused")
     @Subscribe(threadMode = ThreadMode.MAIN)
+    public void onSiteRemoved(OnSiteRemoved event) {
+        if (!event.isError()) {
+            getAdapter().loadSites();
+        } else {
+            // shouldn't happen
```

Comment:
As you mentioned, this shouldn't really happen. But in case it does, it probably means there is an sql error. Do you think we should add a log here with the error details, to be able to track down such cases?